src/common/components/outline/Outline.tsx

```diff
@@ -68,14 +77,35 @@ export const Outline: React.FC<Props> = ({
         ))}
       </Accordion>
 
-      <Flex justify={"flex-end"}>
-        <GButton size="xs" type="secondary" onClick={handleExpandAll}>
-          Expand all
-        </GButton>
-        <GButton size="xs" type="secondary" onClick={handleCollapseAll}>
-          Collapse all
-        </GButton>
-      </Flex>
+      {!expandByDefault && (
```

Comment:
should this be a different prop? allowExpand?

Reply:
Done